contracts/SportMarkets/SportAMMRiskManager.sol

```diff
@@ -0,0 +1,206 @@
+        uint marketRisk = riskMultiplierPerMarket[market];
+
+        if (marketRisk == 0) {
+            (uint tag1, ) = _getTagsForMarket(market);
+            uint riskPerTag = riskMultiplierForSport[tag1];
+            marketRisk = riskPerTag > 0 ? riskPerTag : defaultRiskMultiplier;
+        }
+
+        toReturn = marketRisk;
+    }
+
+    function _calculateCapToBeUsed(address market) internal view returns (uint toReturn) {
+        toReturn = capPerMarket[market];
+        if (toReturn == 0) {
+            (uint tag1, uint tag2) = _getTagsForMarket(market);
+            uint capFirstTag = capPerSport[tag1];
+            capFirstTag = capFirstTag > 0 ? capFirstTag : defaultCapPerGame;
+            toReturn = capFirstTag;
+
+            if (tag2 > 0) {
+                uint capSecondTag = capPerSportAndChild[tag1][tag2];
+                toReturn = capSecondTag > 0 ? capSecondTag : capFirstTag / 2;
+            }
+        }
+    }
+
+    function _getTagsForMarket(address _market) internal view returns (uint tag1, uint tag2) {
+        ISportPositionalMarket sportMarket = ISportPositionalMarket(_market);
+        tag1 = sportMarket.tags(0);
+        tag2 = sportMarket.isChild() ? sportMarket.tags(1) : 0;
+    }
+
+    /* ========== CONTRACT MANAGEMENT ========== */
+
+    /// @notice Setting the Cap per spec. market
+    /// @param _markets market addresses
+    /// @param _capPerMarket The cap amount used for the specific markets
+    function setCapPerMarket(address[] memory _markets, uint _capPerMarket) external {
+        require(
+            msg.sender == owner || ISportPositionalMarketManager(manager).isWhitelistedAddress(msg.sender),
+            "Invalid sender"
+        );
+        for (uint i; i < _markets.length; i++) {
+            capPerMarket[_markets[i]] = _capPerMarket;
+            emit SetCapPerMarket(_markets[i], _capPerMarket);
+        }
+    }
+
+    /// @notice Setting the Cap per Sport ID
+    /// @param _sportID The tagID used for sport (9004)
+    /// @param _childID The tagID used for childid (10002)
+    /// @param _capPerChild The cap amount used for the sportID
+    function setCapPerSportAndChild(
+        uint _sportID,
+        uint _childID,
+        uint _capPerChild
+    ) external onlyOwner {
+        capPerSportAndChild[_sportID][_childID] = _capPerChild;
+        emit SetCapPerSportAndChild(_sportID, _childID, _capPerChild);
+    }
+
+    /// @notice Setting the Cap per Sport ID
+    /// @param _sportID The tagID used for each market
+    /// @param _capPerSport The cap amount used for the sportID
+    function setCapPerSport(uint _sportID, uint _capPerSport) external onlyOwner {
+        capPerSport[_sportID] = _capPerSport;
+        emit SetCapPerSport(_sportID, _capPerSport);
+    }
+
+    /// @notice Setting the Cap per game default value
+    /// @param _capPerGame default cap
+    function setDefaultCapPerGame(uint _capPerGame) external onlyOwner {
+        defaultCapPerGame = _capPerGame;
```

Comment:
I generally think we need to change this value from 5000 to 1000.
Also maybe introduce a MaxCap variable e.g. 20k and ensure no value is greater than it.

Reply:
done. put it in script 1K